common/io/src/kseqpp_fasta_parser.cpp

```diff
@@ -23,17 +23,21 @@ namespace claragenomics
 namespace io
 {
 
-FastaParserKseqpp::FastaParserKseqpp(const std::string& fasta_file)
+FastaParserKseqpp::FastaParserKseqpp(const std::string& fasta_file, int min_sequencece_length)
 {
     klibpp::KSeq record;
     klibpp::SeqStreamIn iss(fasta_file.data());
     std::vector<FastaSequence> seqs;
     int total_len = 0;
```

Comment:
`total_len` is not actually being used anywhere, right?

Reply:
good point. done